Pages/wishlist.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Wishlist | Plantly</title>
+
+    <!-- Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Stylesheets -->
+    <link rel="stylesheet" href="https://aceui.netlify.app/aceui-styles.css" />
+    <link rel="stylesheet" href="../Styles/productListing.css" />
+    <link rel="stylesheet" href="../style.css" />
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+
+  <body>
+    <!-- Main Header -->
+    <header class="main-header flex-row px-8">
+      <nav class="nav-bar">
+        <a href="https://plantlybyaish.netlify.app">
+          <h4 class="nav-logo">🌿Plantly</h4>
+        </a>
+        <div class="nav-search">
+          <form class="form-search">
+            <div class="search-container">
+              <input
+                class="search-input"
+                type="search"
+                name="search"
+                placeholder="What are you looking for?"
+              />
+              <button class="btn-search" type="submit">
+                <i
+                  class="fa fa-search fa-lg"
+                  title="Search"
+                  aria-hidden="true"
+                ></i>
+              </button>
+            </div>
+          </form>
+        </div>
+        <ul class="nav-links">
+          <li class="nav-link">
+            <a href="./wishlist.html"
+              ><i class="fa fa-heart fa-lg" aria-hidden="true"></i
+            ></a>
+          </li>
+          <li class="nav-link">
+            <a href="./cart.html"
+              ><i class="fa fa-shopping-cart fa-lg" aria-hidden="true"></i
+            ></a>
+          </li>
+          <li class="nav-link">
+            <a href="./signup.html">Sign up</a>
+          </li>
+          <li class="nav-link">
+            <a href="./login.html"
+              ><button class="btn btn-primary">Login</button></a
+            >
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!--Top Banner-->
+    <section class="sub-section p-2 mb-2"><h5>Wishlist</h5></section>
+
+    <!-- Main Body-->
+    <main class="body-container mx-8">
+      <!--Product Cards-->
+      <section class="my-1">
+        <div class="all-products">
+          <div class="card-product card-vertical">
+            <div class="card-image">
+              <img
```

Comment:
A meaningful `alt` attribute can be provided for `img` tag

Reply:
added `alt` attribute to `img` tag